anms-ui/public/app/components/management/builder/Build.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div>
+    <h5>ARI Builder</h5>
+    <center>
+      Input Style(CBOR Input/ARI Builder)<br>
+      <toggle-button id="tButton" v-model="stringMode" :labels="{ checked: 'String Input', unchecked: 'ARI Builder' }"
```

Comment:
add the cbor string option and also the ability to send cbor to this page from the builder tab